### PR TITLE
Changed version constraint of AWS SDK

### PR DESCRIPTION
**Pull request by hannesvdvreken**
Merge this in when ready

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
     "require": {
         "php": ">=5.4.0",
         "league/flysystem": "~1.0",
-        "aws/aws-sdk-php": "3.*@dev"
+        "aws/aws-sdk-php": "^3.0.0"
```
Comment (GrahamCampbell) [inline composer.json]: Why not `~3.0` to match all our other version constraints?

Comment (hannesvdvreken) [inline composer.json]: More modern, and it's the default now when you do `composer require aws/aws-sdk-php`. And if jeremeamia would push a critical security fix or something then we would only have to update `^3.0.0` to `^3.0.x`.

Comment (hannesvdvreken) [inline composer.json]: ah no, default would be `^3.0` http://semver.mwl.be/#?package=aws%2Faws-sdk-php

Comment (GrahamCampbell) [inline composer.json]: No. `~3.0` is the standard being used in flysystem. It makes no sense to change it when it does the same thing in this case.

Comment (hannesvdvreken) [inline composer.json]: Ok.

I think we'll soon see `^` be used much more often. Ofc gradually, it makes no sense to change all existing version constraints.

**Comment by frankdejonge** [comment]
Replaced by 749a4d587dd2d21329324a0bb52c5abb82b2c6d4

**Comment by hannesvdvreken** [comment]
:+1:

**Pull request closed without merging**